src/verus.pest

```diff
@@ -922,7 +930,7 @@ attr_core = {
     | "#" ~ bang_str? ~ "[" ~ !"verusfmt" ~ meta ~ "]"
 }
 // Two aliases for attr_core, so that we can print them differently
-attr = {
+attr = !{
```

Comment:
Why did this become atomic?  And will doing so cause us to reject attributes (with spaces) that Rust accepts?

Reply:
The `!{...}` explicitly means "[non-atomic](https://pest.rs/book/grammars/syntax.html#non-atomic)"---this breaks it out of any case that might itself be atomic or compound-atomic.  Since `stmt_list` because atomic, we need to make things like `attr`, `stmt`, and `expr` (i.e., the ones _directly_ inside `stmt_list`) explicitly non-atomic to indeed support cases like things with spaces.

Reply:
Ah, sorry, I misread the Pest rules for atomic.